client/lib/feed-stream-store/feed-stream.js

```diff
@@ -417,6 +420,12 @@ export default class FeedStream {
 	}
 
 	filterNewPosts( posts ) {
+		// filter out posts whose only comments are ones we already have
```

Comment:
we monkey patch this method for the conversation store in feed-stream-store/index... You'll want to modify that one instead.